splash/src/main/res/drawable/splash_gradient.xml

```diff
@@ -0,0 +1,11 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+    android:shape="rectangle">
+
+    <gradient
+        android:angle="90"
+        android:endColor="#2ea1a2"
```

Comment:
Can these be switched to `@color/` resources?

Reply:
Extracted to colors.